plugins/de.cau.cs.kieler.klighd/src/de/cau/cs/kieler/klighd/IViewer.java

```diff
@@ -388,7 +388,7 @@ void addViewChangedListener(IViewChangeListener listener,
      * @param diagramElement
      *            a (hierarchic) {@link KNode}
      * @return <code>true</code> if the {@link KNode} <code>diagramElement</code> is expanded,
-     *         <code>true</code> otherwise.
+     *         <code>false</code> otherwise.
```

Comment:
I sthis correct? Why was this not fixed on the master?

Reply:
I guess I forgot to put this on the master branch. This PR will fix that.